lib/core/versions/latest/BatchWriter.ts

```diff
@@ -77,4 +83,10 @@ export default class BatchWriter implements IBatchWriter {
     // Remove written operations from queue after batch writing has completed successfully.
     await this.operationQueue.dequeue(queuedOperations.length);
   }
+
+  private getNumberOfOpsToWrite (valueTimeLock: ValueTimeLockModel | undefined, normalizedFee: number): number {
+    const numberOfOpsAllowed = ValueTimeLockVerifier.calculateMaxNumberOfOpsAllowed(valueTimeLock, normalizedFee);
+
+    return Math.min(numberOfOpsAllowed, ProtocolParameters.maxOperationsPerBatch);
```

Comment:
Should we log some sort of message here if the maxOperationPerBatch is lower than numberOfOpsAllowed so the admin knows when they are "wasting" money in lock?

Reply:
Adding a log statement here.